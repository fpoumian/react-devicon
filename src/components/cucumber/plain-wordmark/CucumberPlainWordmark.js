
import React from 'react'
import PropTypes from 'prop-types'
import SVGInline from 'react-svg-inline'
import iconSVG from './CucumberPlainWordmark.svg'
import { normalizeUnit } from '../../../utils.js'

/** CucumberPlainWordmark */
function CucumberPlainWordmark({width, height}) {
  return (
     <SVGInline
      svg={iconSVG}
      width={normalizeUnit(width)}
      height={normalizeUnit(height)}
    />
  )
}

CucumberPlainWordmark.propTypes = {
  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
}

export default CucumberPlainWordmark
