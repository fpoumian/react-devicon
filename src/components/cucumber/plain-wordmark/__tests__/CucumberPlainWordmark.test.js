import React from 'react'
import renderer from 'react-test-renderer'
import CucumberPlainWordmark from '../index.js'

it('renders correctly', () => {
  const tree = renderer.create(<CucumberPlainWordmark />).toJSON()
  expect(tree).toMatchSnapshot()
})
