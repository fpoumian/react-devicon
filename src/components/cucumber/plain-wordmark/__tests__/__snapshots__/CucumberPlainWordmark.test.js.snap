// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="SVGInline CucumberPlainWordmark"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<svg class=\\"SVGInline-devicon CucumberPlainWordmark-devicon\\" id=\\"Layer_1\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 362.8 362.8\\"><style>.st0{fill:#00a818}</style><title>cucumber-plain-wordmark</title><path id=\\"Logos\\" class=\\"st0\\" d=\\"M315.2 188.5c-3.7 0-7.1 2-8.8 5.3 0-.8.4-1.8.4-2.5v-1.5h-7v34.9h7v-20.1c0-5.6 3.1-9.2 8.5-9.2h1.4v-7h-1.5v.1zM31.8 201.3c-1.8-3.2-5.1-5.2-8.8-5.2-6.1 0-11 5-11 11.1s4.9 11 11 11c3.6.1 7-1.9 8.8-5l6.4 3.8c-3.4 5.1-9.1 8.2-15.2 8.3-9.9.6-18.3-7-18.9-16.9-.6-9.9 7-18.3 16.9-18.9h2c6.2-.1 11.9 3 15.2 8.3l-6.4 3.5zm12.5-11h7.2v19c-.2 4.5 3.3 8.3 7.8 8.4h.4c4.6-.1 8.2-3.9 8.2-8.5v-19h7.2v19.6c.2 8.3-6.4 15.1-14.7 15.3h-.8c-8.3.3-15.2-6.2-15.5-14.4V190.3h.2zm65 11c-1.8-3.2-5.1-5.2-8.8-5.2-6.1 0-11 5-11 11.1s4.9 11 11 11c3.6.1 7-1.9 8.8-5l6.4 3.8c-3.4 5.1-9.1 8.2-15.2 8.3-9.9.6-18.3-7-18.9-16.9-.6-9.9 7-18.3 16.9-18.9h2c6.2-.1 11.9 3 15.2 8.3l-6.4 3.5zm12.5-11h7.2v19c-.2 4.5 3.3 8.3 7.8 8.4h.4c4.6-.1 8.2-3.9 8.2-8.5v-19h7.2v19.6c0 8.6-6.9 15.5-15.5 15.5s-15.5-6.9-15.5-15.5l.2-19.5zm40 12.7c0-8.7 5.6-13.6 12.4-13.6 4.3-.3 8.3 2.1 10.1 6 1.7-4 5.8-6.4 10.2-6 6.7 0 12.4 5 12.4 13.6v21.2h-7V203c.4-3.1-1.7-5.9-4.8-6.4-.3 0-.6-.1-.8-.1-3.5 0-6 2.1-6 7.4V224h-7.2v-20.2c0-5.3-2.5-7.4-6-7.4-3.1 0-5.7 2.5-5.7 5.6 0 .3 0 .6.1.8V224h-7.7v-21zm54.3-22.8h7.2v14.3c2.7-3.3 6.8-5.2 11-5 9.9 0 17.9 8 17.9 17.9 0 9.9-8 17.9-17.9 17.9-4.3.2-8.4-1.7-11-5v4.2h-7.2v-44.3zm17.7 16.1c-6.1-.2-11.2 4.5-11.5 10.6-.2 6.1 4.5 11.2 10.6 11.5s11.2-4.5 11.5-10.6v-.4c.3-5.8-4.2-10.7-10-11-.2-.1-.4-.1-.6-.1zm57.3 20.5c-3.7 5.1-9.5 8.1-15.8 8.3-9.8.4-18.1-7.3-18.5-17.1s7.3-18.1 17.1-18.5h1.4c10.1.4 18 8.9 17.6 19v1.3h-28c.7 5.2 5.1 9 10.3 9 3.9-.1 7.6-2.1 9.8-5.3l6.1 3.3zm-6.7-13.1c-1-4.7-5.2-8-10-8-4.7 0-8.7 3.4-9.6 8h19.6zm63.4-63.2l-1.3-.7-1.4-.7h-.5l-1.2-.5c-2.3-.8-4.7-1.1-7.1-1.1-12.6 0-22.9 10.3-22.9 22.9 0 10.9 7.8 20.4 18.5 22.4v6.6c13.7-2.1 25.8-12.9 26.7-27.2.7-8.7-3.5-17-10.8-21.7zm-16.9 5.6c.8-.1 1.6.2 2.1.9 1.8 2 1.9 4.9.4 7.1-2-.4-3.7-1.9-4.5-3.8-.3-.6-.4-1.2-.4-1.8.1-1.3 1.1-2.3 2.4-2.4zm-6.9 6.8h.4c.7 0 1.4.2 2 .5 1.9.8 3.3 2.4 3.8 4.3-1.7 1.2-4 1.4-5.9.6-.6-.3-1.1-.6-1.7-1-1.1-.7-1.4-2.2-.7-3.3.6-.8 1.4-1.3 2.1-1.1zm2.4 14.1c-.6.3-1.3.5-2 .5-1.4.1-2.6-.8-2.7-2.2-.1-.8.2-1.6.8-2.1.5-.4 1-.8 1.7-1 .7-.3 1.5-.5 2.3-.5 1.3 0 2.5.3 3.6 1-.4 1.9-1.7 3.4-3.5 4.1l-.2.2zm7.7 4.5c-.3.6-.6 1.1-1 1.7-1 1-2.6 1.1-3.7.1-.5-.5-.8-1.2-.8-1.9 0-.6.2-1.2.4-1.8.8-2 2.5-3.4 4.6-3.8 1.1 1.7 1.3 3.8.5 5.6v.1zm3.8-22.8c.3-.6.6-1.1 1-1.7.5-.6 1.3-1 2.2-.9 1.4.1 2.5 1.3 2.5 2.7 0 .6-.2 1.3-.4 1.8-.8 1.9-2.5 3.2-4.5 3.6-1.3-1.7-1.6-3.9-.6-5.8l-.2.3zm1.3 24.2c-1.8-2-1.9-4.9-.4-7.1 2 .4 3.7 1.8 4.6 3.6.3.6.4 1.2.4 1.8-.3 1.5-1.7 2.6-3.3 2.3-.5-.1-.9-.3-1.3-.6zm8.4-5.9c-.7 0-1.4-.2-2-.5-1.9-.8-3.3-2.4-3.8-4.3 1.1-.7 2.3-1.1 3.6-1 .8 0 1.5.2 2.3.5.6.3 1.1.6 1.7 1 1.1.8 1.3 2.3.5 3.4-.5.7-1.3 1.1-2.1 1l-.2-.1z\\"/></svg>",
    }
  }
/>
`;
